spec/Types.tex

```diff
@@ -94,6 +95,20 @@ \subsection{The Void Type}
 There may be storage associated with a value of type \chpl{void}, in which
 case its lifetime obeys the same rules as a value of type \chpl{int}.
```

Comment:
I can't understand why it says this. At the very least, it seems irrelevant, since we don't promise particular padding sizes, etc.

Reply:
I don't get what exactly it implies either.  That sentence has been there since 2011 though.

Reply:
Yeah, Tom H added it, along with documenting `void`. I think we should remove it.

Reply:
OK, I'll remove it.

Reply:
Removed.